yb-voyager/cmd/import.go

```diff
@@ -238,7 +241,15 @@ func registerImportDataToTargetFlags(cmd *cobra.Command) {
 If any table on YugabyteDB database is non-empty, it prompts whether you want to continue the import without truncating those tables; 
 If you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files with upsert mode.
 Note that for the cases where a table doesn't have a primary key, this may lead to insertion of duplicate data. To avoid this, exclude the table using the --exclude-file-list or truncate those tables manually before using the start-clean flag (default false)`)
-	BoolVar(cmd.Flags(), &truncateTables, "truncate-tables", false, "Truncate tables on target YugabyteDB before importing data. Only applicable along with --start-clean true (default false)")
+	BoolVar(cmd.Flags(), &truncateTables, "truncate-tables", false,
+		"Truncate tables on target YugabyteDB before importing data. Only applicable along with --start-clean true (default false)")
+	BoolVar(cmd.Flags(), &enableFastPath, "enable-fast-path", false,
+		"Enable fast path (non-transactional COPY) to improve import performance for colocated tables. Applies only to YugabyteDB target. Not applicable when importing into source/source-replica during fallforward or fallback. (default: false)")
```

Comment:
I think we can skip this `Not applicable when importing into source/source-replica during fallforward or fallback.` as this is in `registerImportDataToTargetFlags` so this flag is only for import data/import data file